hazelcast/src/main/java/com/hazelcast/internal/jmx/ManagementService.java

```diff
@@ -101,10 +103,17 @@ public void destroy() {
         }
     }
 
-    public static void shutdownAll() {
+    public static void shutdownAll(List<HazelcastInstanceProxy> instances) {
+        for (HazelcastInstanceProxy instance : instances) {
+            shutdown(instance.getName());
```

Comment:
I have no idea, can instance name be `null`? In that case `null String` goes here:

```java
    public static String quote(String text) {
        return Pattern.compile("[:\",=*?]")
                .matcher(text)
                .find() ? ObjectName.quote(text) : text;
    }
```

It shouldn't be a problem I guess, also this was the case before, never had any problems apparently.